### PR TITLE
Update some packages

### PR DESCRIPTION
**Pull request by AkgunFatih**

### CODE REVIEW

File: deepin-control-center/package.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ rundeps    :
     - deepin-qt5integration
     - proxychains-ng
 setup      : |
-    %cmake
+    %cmake -DDISABLE_SYS_UPDATE=YES -DCMAKE_INSTALL_PREFIX=/usr -DDISABLE_OPACITY_ANIMATION=NO -DDISABLE_CLOUD_SYNC=YES -DDCC_DISABLE_GRUB=YES
```
Comment (Justinzobel) [inline deepin-control-center/package.yml]: Disalbing GRUB managment? Solus does use GRUB for legacy boots. It might be worth amending this.

File: deepin-desktop-schemas/files/swap-sched.patch [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+diff --git a/schemas/com.deepin.dde.startdde.gschema.xml b/schemas/com.deepin.dde.startdde.gschema.xml
+index 029e500..9b622c2 100644
+--- a/schemas/com.deepin.dde.startdde.gschema.xml
++++ b/schemas/com.deepin.dde.startdde.gschema.xml
+@@ -18,7 +18,7 @@
+         </key>
+
+         <key type="b" name="swap-sched-enabled">
+-            <default>false</default>
++            <default>true</default>
```
Comment (Justinzobel) [inline deepin-desktop-schemas/files/swap-sched.patch]: What is this change for and source please?

File: deepin-control-center/package.yml (hunk 1 above)
Comment (AkgunFatih) [reply deepin-control-center/package.yml]: oops

File: deepin-desktop-schemas/package.yml [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 source     :
     - git|https://github.com/linuxdeepin/deepin-desktop-schemas.git : 3.13.9
 license    : GPL-3.0-or-later
 component  : desktop
 summary    : GSettings Deepin desktop-wide schemas
 description: |
     GSettings Deepin desktop-wide schemas
+setup      : |
+    # disable swap-sched
+    %patch -Rp1 -i $pkgfiles/bf0c4e43f6b6d508ddd346c2d1e865dae9ae947d.patch
```
Comment (Justinzobel) [inline deepin-desktop-schemas/package.yml]: The patch name isn't consistent with the name of the file above.

File: deepin-gir-generator/files/0001-Makefile-Make-directory.patch [hunk 4]
```diff
@@ -1,30 +0,0 @@
...
- 	cp -r  lib.in/gio-2.0/*       $(OUT_GIR_DIR)gio-2.0
-+	mkdir -p $(OUT_GIR_DIR)glib-2.0
- 	cp -r  lib.in/glib-2.0/*      $(OUT_GIR_DIR)glib-2.0
-+	mkdir -p $(OUT_GIR_DIR)gudev-1.0
- 	cp -r  lib.in/gudev-1.0/*     $(OUT_GIR_DIR)gudev-1.0
- 
- glib-2.0: lib.in/glib-2.0/glib.go.in lib.in/glib-2.0/config.json generator
--- 
-2.20.1
-
```
Comment (Justinzobel) [inline deepin-gir-generator/files/0001-Makefile-Make-directory.patch]: Reason for deletion?

File: deepin-gir-generator/files/glib-2.63.patch [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+@@ -222,7 +222,8 @@
+ 			"launch_uris_as_manager_with_fds"
+ 		],
+ 		"Cancellable": [
+-			"connect"
++			"connect",
++			"source_new"
+ 		],
+ 		"Application": [
+ 			"get_dbus_connection"
```
Comment (Justinzobel) [inline deepin-gir-generator/files/glib-2.63.patch]: Source/reason for patch?

File: deepin-control-center/package.yml (hunk 1 above)
Comment (AkgunFatih) [reply deepin-control-center/package.yml]: what about -DDISABLE_SYS_UPDATE=YES ?

**Comment by Justinzobel** [comment]
Can you please add reason/sources for patches and reasoning for other changes, thanks. Great work though so far!

File: deepin-desktop-schemas/files/swap-sched.patch (hunk 2 above)
Comment (AkgunFatih) [reply deepin-desktop-schemas/files/swap-sched.patch]: from here https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/deepin-desktop-schemas

File: deepin-control-center/package.yml (hunk 1 above)
Comment (Justinzobel) [reply deepin-control-center/package.yml]: Disabling system updates is fine as it likely only supports apt. If you want to submit a patch to DDE upstream to support eopkg that would be great otherwise I'm happy to disable it as Solus does have it's own update checking mechanisms.

File: deepin-gir-generator/files/0001-Makefile-Make-directory.patch (hunk 4 above)
Comment (AkgunFatih) [reply deepin-gir-generator/files/0001-Makefile-Make-directory.patch]: this fixes the building of deepin-api

**Comment by AkgunFatih** [comment]
i am trying to follow the packages here https://www.archlinux.org/packages/?sort=&q=deepin&maintainer=&flagged=

i have used deepin on arch and it works great

File: deepin-desktop-schemas/files/swap-sched.patch (hunk 2 above)
Comment (Justinzobel) [reply deepin-desktop-schemas/files/swap-sched.patch]: I use Arch's pkgbuild's often but what does it do?

File: deepin-turbo/package.yml [hunk 6]
```diff
@@ -11,9 +11,9 @@ description: |
     Applauncherd is a daemon that helps to launch applications faster by preloading dynamically linked libraries and caching stuff. It also saves memory, because all launched applications share certain resources.
     we built a booster type called dtkwidget-booster in this project to support launching dtk-based Apps faster.
 builddeps  :
-    - pkgconfig(qtermwidget5)
```
Comment (Justinzobel) [inline deepin-turbo/package.yml]: Why was this removed?

File: deepin-turbo/package.yml [hunk 7]
```diff
@@ -11,9 +11,9 @@ description: |
...
+    - qt5-tools-devel
```
Comment (Justinzobel) [inline deepin-turbo/package.yml]: Should use pkgconfig where possible ie pkgconfig(Qt5UiTools)

File: deepin-wallpapers/package.yml [hunk 8]
```diff
@@ -18,3 +18,9 @@ install    : |
 
     install -dm0755 "${installdir}/var/cache"
     cp -r image-blur "${installdir}/var/cache/"
+
+    # Suggested by upstream
```
Comment (Justinzobel) [inline deepin-wallpapers/package.yml]: Link to suggestion if it was suggested or sourced from somewhere please. :)

File: deepin-desktop-schemas/files/swap-sched.patch (hunk 2 above)
Comment (AkgunFatih) [reply deepin-desktop-schemas/files/swap-sched.patch]: i dont have any clue :D

File: deepin-turbo/package.yml (hunk 6 above)
Comment (AkgunFatih) [reply deepin-turbo/package.yml]: [BuildDep] Checking build-deps for deepin-turbo-0.0.3-1
[pkgconfig:qtermwidget5] This dependency is not in any repo
[BuildDep] pkgconfig(qtermwidget5) build dep does not exist in the repository.
and its not needed anyway

**Comment by Justinzobel** [comment]
Why did you close it? It's good work.
